src/mgmt/srv_internal.h

```diff
@@ -98,6 +98,14 @@ int ds_mgmt_destroy_pool(uuid_t pool_uuid, const char *group, uint32_t force);
 void ds_mgmt_hdlr_pool_create(crt_rpc_t *rpc_req);
 void ds_mgmt_hdlr_pool_destroy(crt_rpc_t *rpc_req);
 int ds_mgmt_pool_get_acl(uuid_t pool_uuid, struct daos_acl **acl);
+/* list pools:
+ * Note: ds_mgmt_list_pools poolsp arg is a CaRT structure type
+ */
+void
+free_mgmt_list_pools(struct mgmt_list_pools_one **poolsp, uint64_t len);
```

Comment:
If this is going public (even internally) you'll probably want to follow the naming convention, i.e. ds_mgmt_free_pool_list